bin/spack

```diff
@@ -142,6 +146,13 @@ def _main(args, unknown_args):
         import spack.util.debug as debug
         debug.register_interrupt_handler()
 
+    if args.color and args.no_color:
+        tty.die("Do not set and cancel color")
+    if args.color:
+        tty.set_color(True)
+    elif args.no_color:
+        tty.set_color(False)
```

Comment:
What about `not args.color and not args.no_color`? What is the default? You can actually make these flags mutually exclusive. See `spack edit -c edit` for an example.